pandas/core/generic.py

```diff
@@ -802,12 +805,27 @@ def swaplevel(self, i=-2, j=-1, axis=0):
         0  1  4
         1  2  5
         2  3  6
+
+        Using axis-style parameters
```

Comment:
are you showing examples of both? maybe put your highly recommended warning here?

Reply:
Yep, added.